bundler/lib/bundler/templates/newgem/README.md.tt

```diff
@@ -6,15 +6,9 @@ TODO: Delete this and the text above, and describe your gem
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add this line to your application's Gemfile by executing:
```

Comment:
`bundle add` not only adds the gem to the `Gemfile`, but also installs it. Can we clarify that somehow?

Reply:
I changed to `Add this line to your application's Gemfile and install by executing:`

Reply:
Actually I went a step further and attempted to describe the difference between installing via bundler vs gem. I also removed *your application* since it may not actually be the persons' application.

```md
## Installation

Install the gem and add to the application's Gemfile by executing:

    $ bundle add <%= config[:name] %>

If bundler is not being used to manage dependencies, install the gem by executing:

    $ gem install <%= config[:name] %>
```